electron/scripts/lib.js

```diff
@@ -3,8 +3,8 @@ var BASE_URL = "http://127.0.0.1:5000/";
 // Get the query parameters of the current page (not visable in electron)
 function getParameterByName(name, url) {
 	var regex = new RegExp("[?&]" + name + "=([^&#]*)(&|$|#)");
```

Comment:
I feel like there must be a library that does this already, but I don't know that for a fact.